src/app/+run-tale/run-tale/run-tale.component.html

```diff
@@ -1,130 +1,133 @@
+                        <div class="tale-info">
+                            <div class="ui stackable grid">
+                                <div class="ten wide column">
+                                    <div class="title-info">
+                                        <div class="img-holder">
+                                            <img [src]="tale.illustration" *ngIf="tale.illustration">
+                                        </div>
+                                        <img [src]="tale.icon" class="env" *ngIf="tale.icon">
+                                        <div class="title">{{ tale.title | truncate:100 }}</div>
+                                        <div class="author">
+                                            <span *ngFor="let author of tale.authors; index as i; trackBy: trackByAuthorOrcid">
+                                                <span *ngIf="i > 0">, </span> <span [title]="author.orcid">{{ author.firstName }} {{ author.lastName }}</span>
+                                            </span>
+                                            <span *ngIf="!tale.authors.length && creator">{{ creator.firstName }} {{ creator.lastName }}</span>
+                                            <span *ngIf="!tale.authors.length && !creator">???</span>
+                                        </div>
+                                        <div class="title-tags">
+                                            <span class="ui horizontal label wt-green" *ngIf="tale.copyOfTale">COPY</span>
+                                            <span class="ui horizontal label wt-green">{{ tale.public ? 'PUBLIC' : 'PRIVATE'}}</span>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="six wide column" style="text-align: right">
+                                    <app-tale-run-button class="tale-run-button" [tale]="tale" [instance]="instance" isPrimary="true" (taleInstanceStateChanged)="taleInstanceStateChanged($event)"></app-tale-run-button>
+                                    <a [routerLink]="dashboardLink" routerLinkActive="active" class="ui button" style="margin-right: 10px">
+                                        Close
+                                    </a>
+
+                                    <div id="tale-actions-dropdown"  class="ui floating tale dropdown theme" tabindex="0">
+
+                                        <mat-icon fontSet="fa" fontIcon="fa-ellipsis-v"></mat-icon>
+
+                                        <div class="menu transition hidden" tabindex="-1">
+                                            <div class="item" (click)="rebuildTale()" *ngIf="isTaleWritable()"><i class="fas fa-fw fa-redo-alt"></i> Rebuild Tale</div>
+                                            <div class="item" (click)="restartTale()" *ngIf="isTaleWritable() && instance && instance.status === -1">

```

Comment:
```suggestion
                                            <div class="item" (click)="restartTale()" *ngIf="isTaleWritable() && instance && instance.status === 1">

```

Shouldn't this be 1?

Reply:
Good catch! Indeed, this should probably be a 1

This is why I shouldn't make PRs at 3AM :X lol